templates/web/fixmystreet.com/report/new/extra_name.html

```diff
@@ -0,0 +1,40 @@
+[%# Same as parent, with added gender question %]
+
+[% IF c.cobrand.ask_gender_question %]
+<p>
+We’re conducting research to make sure our site is good for everyone;
+please help us out by providing your gender:
```

Comment:
We should add a note making it clear that their answer will not be shared with the council or displayed publicly. Perhaps a link to a longer explanation of why we're asking this on the FAQ or privacy page?